pennylane/debugging/debugger.py

```diff
@@ -132,33 +132,198 @@ def pldb_device_manager(device):
 
 
 def breakpoint():
-    """Launch the custom PennyLane debugger."""
+    """A function which freezes execution and launches the PennyLane debugger (PLDB).
+
+    This function marks a location in a quantum circuit (QNode). When it is encountered during
+    execution of the quantum circuit, an interactive debugging prompt is launched to step
+    throught the circuit execution using Pdb like commands (:code:`list`, :code:`next`,
+    :code:`continue`, :code:`quit`).
```

Comment:
Can we link to a central page that lists the commands?

Reply:
I added a link to the `qml.debugging` api page.